jib-core/src/main/java/com/google/cloud/tools/jib/docker/json/DockerLoadManifestTemplate.java

```diff
@@ -29,21 +28,21 @@
  * <p>Example manifest JSON:
  *
  * <pre>{@code
- * [{
+ * {
```

Comment:
I'll rename the class to `DockerLoadManifestEntryTemplate` and update the Javadoc. Then I don't think we'll need `DockerLoadManifestTemplate` anymore, since it'd be just `List<DockerLoadManifestEntryTemplate>`.